src/cmd/tools.go

```diff
@@ -95,13 +95,33 @@ var k9sCmd = &cobra.Command{
 	},
 }
 
+var clearCacheCmd = &cobra.Command{
+	Use:     "clear-cache",
+	Aliases: []string{"c"},
+	Short:   "Clears the configured git and image cache directory",
+	Run: func(cmd *cobra.Command, args []string) {
+		if zarfCache != config.ZarfDefaultCachePath && cachePathClean(zarfCache) {
```

Comment:
Since the `zarfCache` variable has the default path set by cobra, I think this function can get simplified down to 

```Go
if cachePathClean(zarfCache) {
    if err := os.RemovedAll(cachePath); err != nil {
        message.Fatalf(.....)
    }
}
```

Reply:
I'll simplify it but also need to test that special urls don't mess it up (there is ~ handling in those other spots)

Reply:
It le worked